src/components/victory-area/victory-area.js

```diff
@@ -283,7 +284,23 @@ export default class VictoryArea extends React.Component {
       PropTypes.string,
       PropTypes.arrayOf(PropTypes.string),
       PropTypes.arrayOf(PropTypes.func)
-    ])
+    ]),
+    /**
+     * The containerComponent prop takes an entire component which will be used to surround
```

Comment:
see comment in VictoryPie about description wording. Make sure the description matches the component _i.e._ reference VictoryArea here instead of VictoryChart